docs/docs/docs/guides/usage-with-react-navigation.mdx

```diff
@@ -216,7 +216,13 @@ function MyTabs() {
   );
 }
 ```
+#### `renderBottomAccessoryView` <Badge text="iOS" type="info" />
 
+Function that returns a React element to render as [bottom accessory](https://developer.apple.com/documentation/uikit/uitabbarcontroller/bottomaccessory).
+
+:::note
+This feature requires iOS 26.0 or later and is only available on iOS. On older versions, this prop is ignored.
+:::
```

Comment:
I think for react navigation integration it might make more sense to put it into options, so users can render different accessory view per screen. Should be fairly simple change